anyblok_furetui/furetui/furetui.py

```diff
@@ -198,3 +198,36 @@ def apply_value(value):
             res = getattr(obj, call)(**options, **data)
 
         return res
+
+    @classmethod
+    def validate_resources(cls):
+        res = []
+        res.extend(cls.validate_form_resources())
+        res.extend(cls.validate_list_resources())
+        return res
+
+    @classmethod
+    def validate_form_resources(cls):
+        res = []
+        Form = cls.registry.FuretUI.Resource.Form
+        for resource in Form.query().filter(Form.template.isnot(None)):
+            try:
+                resource.get_definitions()
+            except Exception as e:
```

Comment:
I'm wondering what's the best should we catch all Exception or create a dedicated exception and only catch them ?

Reply:
Yes I will do that

Reply:
done